lib/adal-angular.js

```diff
@@ -327,8 +327,17 @@
 
                 if ($injector.has('$transitions')) {
                     var $transitions = $injector.get('$transitions');
-                    $transitions.onStart({}, (transition) => stateChangeHandler(null, transition.to(), transition.params('to'), transition.from(), transition.params('from')));
-                    $transitions.onError({}, (transition) => stateChangeErrorHandler(null, transition.to(), transition.params('to'), transition.from(), transition.params('from'), transition.error()));
+
+                    function onStartStateChangeHandler(transition) {
+                        stateChangeHandler(null, transition.to(), transition.params('to'), transition.from(), transition.params('from'))
```

Comment:
Nitpicking: would you mind adding a semicolon at the end of this line? Same applies to line 336 too.